MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+
+#### Bootstrapping the Process
+
+However, due to the limited current capacity we propose that the Mina Foundation, in collaboration with o1 Labs and other members of the Mina ecosystem, will temporarily fill these new roles as needed while the processes are tested, and the capacity and expertise grows in the community.
+
+This bootstrapping period would last no longer than 12 months.
+
+### Introducing Mass Deliberation Tools and Public Debates
+
+We propose to implement **new tools** to collect and analyze the comments and sentiments from the community at different phases in the MIP process. These include:
+
+- Polling on MinaResearch. The community is encouraged to experiment with different methods to increase effectiveness.
+- Survey bot on Discord. The development and usage of open source AI bots on Discord to gather, organize and summarize mass feedback. The community is encouraged to review and propose improvements to the development:
+https://github.com/MinaFoundation/gptSurveySummarizer 
+- On-Chain Voting. The community is encouraged to suggest any improvements that could be proposed in a future MIP:
+https://github.com/MinaFoundation/mina-on-chain-voting/tree/main 
+https://github.com/MinaFoundation/govbot 
+- Governance Dashboards. The community is encouraged to experiment with different methods and tools to increase participation in the MIP process.
+https://minascan.io/mainnet/governance/MIPs 
+
+We propose to introduce new types of public debates:
+
+- During the expert part of the Review Phase: The MIP Facilitator will schedule, livestream and record open and public meetings between the MIP Author, the MIP Review Group and other stakeholders. The meeting minutes will be posted and shared in the designated channels for public consumption, allowing the community members who are interested to follow the process and contribute with comments. 
+- During the community part of the Review Phase: There will be a period of time for the community to carefully read and consider the MIP and the Review Group’s Risk Assessment. The MIP Facilitator will coordinate with the Mina Foundation to organize a community call where the MIP Author and the MIP Review Group can explain the MIP, including its risks, benefits and tradeoffs, in a way that is comprehensible to non-expert audiences and respond to community questions. Dates for the on-chain vote will be set, along with instructions on how to participate.
+
+### Changes in the Drafting and Review Phases
+
+**Drafting Phase - MIP Submission**
+
+During the Drafting phase, when the MIP Author submits a new MIP as a pull request in GitHub, a MIP Facilitator is assigned to assist the MIP Author, review the MIP and (if necessary, consult with the MIP Reviewers to) decide whether to approve or reject the pull request according to the following acceptance criteria:
```

Comment:
How is a facilitator assigned or selected?

Reply:
This needs to be defined before I would consider this ready for a vote.

Reply:
My initial thought was to use a random assignment process for Facilitators to mitigate risks of potential bias or complicity between the MIP Author and the Facilitator. However, I’m also exploring the idea of collective decision-making, where key decisions (such as approving a new MIP for review) are made collaboratively by the entire group of Facilitators. This would add an additional layer of security and accountability.

In any case, I still think at least one Facilitator should be designated as the main point of contact for the MIP Author to ensure responsiveness and continuity throughout the process. Any thoughts on these ideas or other suggestions to further enhance impartiality and efficiency in Facilitator assignments?